core/src/main/java/eu/fasten/core/data/metadatadb/MetadataDao.java

```diff
@@ -1067,20 +1067,22 @@ public String getModuleCallables(String packageName,
             return null;
         }
 
+        GsonBuilder builder = new GsonBuilder();
+        Gson gson = builder.create();
+
         // Go through each callable, parse fasten uri, insert signature.
         for (Object j : json) {
-            JSONObject jObj = (JSONObject) j;
-            var uri = jObj.getString("fasten_uri");
-
             try {
-                var uriObject = FastenUriUtils.parsePartialFastenUri(uri);
-                jObj.put("method_name", uriObject.get(2));
```

Comment:
What would be the side effect of removing these two fields, i.e., `method_name`? Maybe the partners are using them.

Reply:
The field was included by me for web use. Not sure if any of the partners are using it. Tbh, I doubt it. The value can still be found in the JSON, but just not by `method_name`, but in `fasten_uri/functionOrAttributeName`.